static/app/views/onboarding/targetedOnboarding/onboarding.tsx

```diff
@@ -80,11 +99,33 @@ function Onboarding(props: Props) {
     browserHistory.replace(`/onboarding/${props.params.orgId}/${previousStep.id}/`);
   };
 
+  const genSkipOnboardingLink = () => {
+    const source = `targeted-onboarding-${stepId}`;
+    return (
+      <SkipOnboardingLink
+        onClick={() =>
+          trackAdvancedAnalyticsEvent('growth.onboarding_clicked_skip', {
+            organization,
+            source,
+          })
+        }
+        to={`/organizations/${organization.slug}/issues/`}
+      >
+        {t('Skip Onboarding')}
+      </SkipOnboardingLink>
+    );
+  };
+
   return (
     <OnboardingWrapper data-test-id="targeted-onboarding">
-      <SentryDocumentTitle title={t('Welcome')} />
+      <SentryDocumentTitle title={stepObj.title} />
       <Header>
         <LogoSvg />
+        <ProgressBar>
```

Comment:
Can you please leave the ProgressStep out from this PR? I think we've got a different component for it now. I'm working on it now and this could introduce merge conflicts down the road.

Reply:
@Neo-Zhixing right, I just removed it